### ISSUE TITLE
Value of weighting factor

### ISSUE THREAD

**Issue by NKSagarReddy** [snap-research/SnapFusion#3]
Hi, Thank you for a great contribution through this paper!
This paper provided more clarity over step distillation compared to existing works in Progressive Distillation and w-guided models using ablation studies.

I had one doubt in the Loss Function which you have mentioned in the paper,
The Hyperparameter, Gamma, which is a weighting factor between distillation loss and original loss, Could you please mention the value of the weighting factor which you have used? I have tried looking through the paper, but I couldn't find it.

**Comment by NKSagarReddy** [comment]
Vanilla distillation is working well but I am unable to reproduce CFG distillation effectively to get visually appealing results.

Hoping you could publish more info on hyperparameters of the v-prediction finetuning and CFG distillation.
